### PR TITLE
BLE Operations called from calling thread

### PR DESCRIPTION
**Pull request by philips77**
Calling BLE operations from any thread, not necessarily main.
Delegates will be called using handler specified in the constructor.

See also #133, #135.

### CODE REVIEW

File: ble/src/main/java/no/nordicsemi/android/ble/BleManager.java [hunk 1]
```diff
@@ -1910,7 +1903,7 @@ private boolean internalRequestConnectionPriority(@ConnectionPriority final int
 	 * Enqueues a request to set the preferred PHY.
 	 * <p>
 	 * PHY LE 2M and PHY LE Coded are supported only on Android Oreo or newer.
-	 * You may safely request other PHYs on older platforms, but  you will get PHY LE 1M
+	 * You may ly request other PHYs on older platforms, but  you will get PHY LE 1M
```
Comment (alesrazym) [inline ble/src/main/java/no/nordicsemi/android/ble/BleManager.java]: Probably unintended

File: ble/src/main/java/no/nordicsemi/android/ble/BleManager.java [hunk 2]
```diff
@@ -1952,7 +1944,7 @@ private boolean internalSetPreferredPhy(@PhyMask final int txPhy, @PhyMask final
 	 * Reads the current PHY for this connections.
 	 * <p>
 	 * PHY LE 2M and PHY LE Coded are supported only on Android Oreo or newer.
-	 * You may safely read PHY on older platforms, but you will get PHY LE 1M as TX and RX PHY
+	 * You may ly read PHY on older platforms, but you will get PHY LE 1M as TX and RX PHY
```
Comment (alesrazym) [inline ble/src/main/java/no/nordicsemi/android/ble/BleManager.java]: Probably unintended

Comment (philips77) [reply ble/src/main/java/no/nordicsemi/android/ble/BleManager.java]: Thanks for the review.

**Comment by philips77** [comment]
@alesrazym Your issues have been fixed in #140.

**Pull request merged**